client/src/services/instances.js

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import axios from './axios';
+
+const getInstances = async (courseId) => {
```

Comment:
`getSisuInstances` might be a more appropriate name, since we will also need to get course instances from our own database.